<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregarHotel</class>
 <widget class="QDialog" name="agregarHotel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre de hotel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_nombreHotel"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Direcci√≥n</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_dir">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Cantidad de habitaciones</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_cantHabitaciones"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Precio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_precio"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="agregar">
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit_nombreHotel</zorder>
  <zorder>lineEdit_dir</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_cantHabitaciones</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_precio</zorder>
  <zorder>agregar</zorder>
  <zorder>cancelar</zorder>
  <zorder>cancelar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
